src/Whip_WPDismissOption.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Represents the WordPress option for saving the dismissed messages.
+ */
+class Whip_WPDismissOption implements Whip_DismissStorage {
+
+	/** @var string */
+	protected $option_name = 'whip_dismissed_for_wp_version';
+
+	/**
+	 * Saves the value to the options.
+	 *
+	 * @param string $dismissedVersion The value to save.
+	 *
+	 * @return bool True when successfull.
+	 */
+	public function set( $dismissedVersion ) {
+		return update_option( $this->option_name, $dismissedVersion );
```

Comment:
`update_option()` only returns `true` if the value was indeed changed. If the value is already set to what you want it to update to, the call is to be considered "successful", but `update_option()` will return `false`.
Probably only a theoretical problem, though, as this method should only be called if the option has not been set to the correct version befor.